src/interp.cc

```diff
@@ -2903,6 +2909,22 @@ Result Thread::Run(int num_instructions) {
       case Opcode::F64X2Sqrt:
         CHECK_TRAP(SimdUnop<v128, int64_t>(FloatSqrt<double>));
         break;
+
+      case Opcode::F32X4ConvertSI32X4:
+        CHECK_TRAP(SimdUnop<v128, int32_t>(SimdConvert<float, int32_t>));
+        break;
+
+      case Opcode::F32X4ConvertUI32X4:
+        CHECK_TRAP(SimdUnop<v128, uint32_t>(SimdConvert<float, uint32_t>));
+        break;
+
+      case Opcode::F64X2ConvertSI64X2:
+        CHECK_TRAP(SimdUnop<v128, int64_t>(SimdConvert<double, int64_t>));
+        break;
+
+      case Opcode::F64X2ConvertUI64X2:
+        CHECK_TRAP(SimdUnop<v128, uint64_t>(SimdConvert<double, uint64_t>));
```

Comment:
This case uses `wabt_convert_uint64_to_double` to handle a floating point rounding issue on windows. Can you do the same here?

Reply:
Done.